includes/wikia/GlobalFunctions.php

```diff
@@ -916,19 +916,19 @@ function wfTimeFormatAgo( $stamp, $hideCurrentYear = true ) {
 		// format is date, with a year (July 26, 2008)
 		$res = $wgLang->date( wfTimestamp( TS_MW, $stamp ) );
 	} elseif ( $ago < 60 ) {
-		// Under 1 min: to the second (ex: 30 seconds ago)
+		// Under 1 min: to the second (ex: 30s)
 		$res = wfMsgExt( 'wikia-seconds-ago', array( 'parsemag' ), $ago );
 	}
 	else if ( $ago < 3600 ) {
-		// Under 1 hr: to the minute (3 minutes ago)
+		// Under 1 hr: to the minute (3m)
 		$res = wfMsgExt( 'wikia-minutes-ago', array( 'parsemag' ), floor( $ago / 60 ) );
 	}
 	else if ( $ago < 86400 ) {
-		// Under 24 hrs: to the hour (4 hours ago)
+		// Under 24 hrs: to the hour (4h)
 		$res = wfMsgExt( 'wikia-hours-ago', array( 'parsemag' ), floor( $ago / 3600 ) );
 	}
-	else if ( $ago < 30 * 86400 ) {
-		// Under 30 days: to the day (5 days ago)
+	else if ( $ago < 7 * 86400 ) {
+		// Under 7 days: to the day (5d)
 		$res = wfMsgExt( 'wikia-days-ago', array( 'parsemag' ), floor( $ago / 86400 ) );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)